cardano-db-sync/src/Cardano/DbSync/Fix/PlutusScripts.hs

```diff
@@ -147,7 +147,7 @@ scrapScriptBlock cblk = case cblk of
   BlockShelley _ -> error "No Plutus Scripts in Shelley"
   BlockAllegra _ -> error "No Plutus Scripts in Allegra"
   BlockMary _ -> error "No Plutus Scripts in Mary"
-  _ -> mempty -- error "TODO: Conway not supported"
+  _ -> mempty -- This bug existed in a version that didn't support Conway
```

Comment:
is it worth being explicit? by having both:
```
BlockConway _ -> mempty
_ -> mempty -- This bug existed in a version that didn't support Conway
```

Reply:
I don't think this would work, as the _ would raise an Unreachable case warning. The current approach is also future proof, maybe the comment can also become 
```suggestion
  _ -> mempty -- This bug existed in a version that didn't support Conway or later eras
```

Reply:
That works for me. I was unsure without going to the library what other "Block" types there were, thanks for explaining.